### PR TITLE
feat: add auditing describe

### PR DESCRIPTION
**Pull request by gssbzn**
## Proposed changes

Add auditing describe 

## Further comments

```bash
./bin/atlas auditing describe 
AUDIT AUTHORIZATION SUCCESS   AUDIT FILTER   CONFIGURATION TYPE   ENABLED
false                                        NONE                 false
```

### CODE REVIEW

File: internal/cli/atlas/auditing/describe.go [hunk 1]
```diff
@@ -0,0 +1,87 @@
+// Copyright 2020 MongoDB Inc
```
Comment (tibulca) [inline internal/cli/atlas/auditing/describe.go:1]:
```suggestion
// Copyright 2023 MongoDB Inc
```

File: internal/cli/atlas/auditing/auditing.go [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+package auditing
+
+import (
+	"github.com/spf13/cobra"
+)
+
+func Builder() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "auditing",
+		Short: "Returns and edits database auditing settings for MongoDB Cloud projects.",
```
Comment (tibulca) [inline internal/cli/atlas/auditing/auditing.go:24]:
```suggestion
		Short: "Returns database auditing settings for MongoDB Cloud projects.",
```
maybe add back "and edits" when those edit commands are added?

File: test/e2e/atlas/auditing_test.go [hunk 3]
```diff
@@ -0,0 +1,48 @@
+// Copyright 2021 MongoDB Inc
```
Comment (tibulca) [inline test/e2e/atlas/auditing_test.go:1]:
```suggestion
// Copyright 2023 MongoDB Inc
```

**Review by tibulca — APPROVED** [review]

**Review by sarahsimpers — APPROVED** [review]
LGTM

**Review by colm-quinn — APPROVED** [review]

**Pull request merged**